Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'jquery-ui-rails'
 # Our database is MongoDB
 gem 'mongoid', '>= 6.2.0', '< 7'
 gem 'bson_ext'
-gem 'mongoid-history', '0.6.1' # gives us object history
+gem 'mongoid-history', '< 1.0' # gives us object history
```

Comment:
There isn't a version higher than `0.8.1` for this library, so you might need this level of safety on the semver. Nothing wrong with what you did, just figured I'd point that out.